auto_py_to_exe/web/js/i18n.js

```diff
@@ -825,6 +828,9 @@ const translationMap = {
                 ru: 'Будьте внимательны при использовании дополнительных файлов в режиме одного файла;\n' +
                     '<a href="https://stackoverflow.com/a/13790741/" style="text-decoration: none;">Прочитайте Это</a>\n' +
                     'и обновите свой код для работы с PyInstaller.',
+                de: 'Vorsicht bei Verwendung von zusätzlichen Dateien im Modus _Eine Datei_;\n' +

```

Comment:
@ackhh - do we need the underscores in this? "_"

Reply:
I wanted to quote "Eine Datei" somehow, wasn't sure whether to use " or any other character. It would be okay without any quotation at all but "... im Modus Eine Datei " (no quoting at all) doesn't feel quite right.

Reply:
All good, I can quote it 👍